### PR TITLE
TGL-RVP: add definitions for RT711 (SDW) and RT1308 (SDW)

### PR DESCRIPTION
**Pull request by RanderWang**
RT711 is on link0 and two RT1308s on link1

### CODE REVIEW

**Comment by RanderWang** [comment]
@plbossart  There is acpi table for two sdw 1308s.

**Comment by lgirdwood** [comment]
@RanderWang can you fix the conflicts. Thanks

File: TGL-RVP/rtk_sdw_link.asl [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+   2,
+  "INTEL",
+  "RTKSDWTabl",
+  0x1000
+  )
+{
+  External(\_SB.PC00.HDAS.SNDW, DeviceObj)
+
+  Scope (_SB.PC00.HDAS.SNDW) {
+        Device (RTK0) // RT711 on link1
```
Comment (plbossart) [inline TGL-RVP/rtk_sdw_link.asl]: link0?

**Review by plbossart — CHANGES_REQUESTED** [review]
typo below?

Also I am not sure why we have conflicts, but you may want to call this rtk_sdw_711_1308 and have a different file for the rtk_sdw_711_i2s_1308 case?

**Comment by plbossart** [comment]
TGL-RVP/rtk_sdw_link.asl     20:   "RTKSDWTabl",
Error    6155 -         Invalid OEM Table ID ^  (Length cannot exceed 8 characters)

**Comment by RanderWang** [comment]
> TGL-RVP/rtk_sdw_link.asl 20: "RTKSDWTabl",
> Error 6155 - Invalid OEM Table ID ^ (Length cannot exceed 8 characters)

I validated it on my TGL-U, I don't know why it doesn't work on TGL-Y

**Comment by plbossart** [comment]
> > TGL-RVP/rtk_sdw_link.asl 20: "RTKSDWTabl",
> > Error 6155 - Invalid OEM Table ID ^ (Length cannot exceed 8 characters)
> 
> I validated it on my TGL-U, I don't know why it doesn't work on TGL-Y

the error happens during the .aml file generation, so it's possible that you are using different acpica-tools. it's not related to the hardware proper. I called it RSDWTABL and things worked fine.

**Review by plbossart — APPROVED** [review]

**Pull request merged**